### PR TITLE
KS-101, KS-105

### PR DESCRIPTION
**Pull request by leykov**
* KS-101: adding custom http header for merchant ID
* KS-105: debug logging the elapsed time for a request
* KS-105: improved log messages time format
*Sample*:
2017-06-01T14:54:47.993+03:00 [DEBUG] Kount.Ris.Request - MERC = 999666 SESS = 5d3c29c8d29f4deaa2d20658c031878b SDK_ELAPSED = 262 ms.

### CODE REVIEW

**Comment by boyankukushev** [comment]
Please, put affected issues in pull request title. Thanks :)

File: SDK/Kount/Ris/Request.cs [hunk 1]
```diff
@@ -229,10 +232,16 @@ public Kount.Ris.Response GetResponse(bool validate = true)
             webReq.Method = "POST";
             webReq.ContentType = "application/x-www-form-urlencoded";
             webReq.ContentLength = buffer.Length;
+            string mercId = this.GetParam("MERC");
+            if (!String.IsNullOrEmpty(mercId))
```
Comment (boyankukushev) [inline SDK/Kount/Ris/Request.cs]: I think `MERC` is a really mandatory field; validation would fail first it it's missing.

Comment (leykov) [inline SDK/Kount/Ris/Request.cs]: **Boyan**, you are right but validation checks for `null` not for `empty`(in .net sdk). I don't know if it's good to set header with empty string.

Comment (boyankukushev) [inline SDK/Kount/Ris/Request.cs]: Try running a request with an empty MERC, it should fail during validation.

**Pull request merged**